Cargo.toml

```diff
@@ -11,4 +11,5 @@ members = [
     "examples/hello_world",
     "examples/hello_router",
     "examples/basic_router",
+    "examples/basic_diesel",
```

Comment:
Suggest we call this example simply "diesel"

Reply:
Indeed. But I still need to name my package in `Cargo.toml` `basic_diesel` or I will get a conflict with the real `diesel` crate.